float64 kp1
float64 kp2
float64 kp3
float64 kd1
float64 kd2
float64 kd3
float64 leg_saturation_cap
float64 hip_saturation_cap
float64 epsilon
float64 torso_offset
float64 s_freq

uint8 s_mode

bool stance_leg
